UPGRADE-4.4.md

```diff
@@ -91,3 +91,8 @@ Validator
 
  * Deprecated passing an `ExpressionLanguage` instance as the second argument of `ExpressionValidator::__construct()`. 
    Pass it as the first argument instead.
+
+Cache
+----------
+
+ * Add new cache adapter to be able using Couchbase as cache system.
```

Comment:
this should be added to the CHANGELOG file of the component instead (this one is for upgrades info)

Reply:
Ok, change done.